test/string_test.dart

```diff
@@ -129,18 +135,15 @@ void main() {
       expect('message digest'.md5, 'f96b697d7cb7938d525a2f31aaf161d0');
       expect('ഐ⌛酪Б👨‍👨‍👧‍👦'.md5, 'c7834eff7c967101cfb65b8f6d15ad46');
       expect(
-          'abcdefghijklmnopqrstuvwxyz'.md5,
-          'c3fcd3d76192e4007dfb496cca67e13b'
-      );
+          'abcdefghijklmnopqrstuvwxyz'.md5, 'c3fcd3d76192e4007dfb496cca67e13b');
       expect(
           'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789'.md5,
-          'd174ab98d277d9f5a5611c2c9f419d9f'
-      );
+          'd174ab98d277d9f5a5611c2c9f419d9f');
       expect(
           '12345678901234567890123456789012345678901234567890123456789012'
-              '345678901234567890'.md5,
-          '57edf4a22be3c955ac49da2e2107b67a'
-      );
+                  '345678901234567890'
+              .md5,
```

Comment:
The formatting does not seem right here...

Reply:
This was generated by the default dart formatter, I had not touched that line.

Reply:
Okay, I'll change this test later because it looks strange...